lib/features/profile/presentation/screens/edit_profile_screen.dart

```diff
@@ -0,0 +1,155 @@
+import 'dart:io';
+import 'package:final_project/core/domain/entities/user.dart';
+import 'package:final_project/core/domain/error/error_toast.dart';
+import 'package:final_project/core/presentation/validation/validators.dart';
+import 'package:final_project/core/presentation/widgets/custom_elevated_button.dart';
+import 'package:final_project/core/presentation/widgets/custom_text_form_field.dart';
+import 'package:final_project/features/profile/presentation/bloc/profile_cubit.dart';
+import 'package:final_project/features/profile/presentation/bloc/profile_state.dart';
+import 'package:final_project/features/profile/presentation/screens/view_profile_screen.dart';
+import 'package:final_project/features/profile/presentation/widgets/edit_profile_image_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:image_picker/image_picker.dart';
+
+class EditProfileScreen extends StatelessWidget {
+  const EditProfileScreen();
+  static const routeName = 'edit_profile';
+  @override
+  Widget build(BuildContext context) {
+    final user = ModalRoute.of(context)!.settings.arguments! as User;
```

Comment:
Never put variables in the build method. You can't know when it will be called again recreating these variables making the previous state obsolete. Use `StateFullWidget` instead.